bin/alluxio

```diff
@@ -29,6 +29,7 @@ function printUsage {
   echo -e "  runMesosTest  \t Test the Alluxio integration with Mesos. Try 'alluxio mesosTest -help' for more help."
   echo -e "                \t NOTE: This command requires Mesos to be running and will stop any Alluxio servers that are currently running."
   echo -e "  readJournal   \t Read an Alluxio journal file from stdin and write a human-readable version of it to stdout."
+  echo -e "  updateJournal \t Update an Alluxio v0 (< 1.5.0) journal to an Alluxio v1 (>= 1.5.0) journal."
```

Comment:
nit: `Update an Alluxio journal version 0 (Alluxio version < 1.5.0) to Alluxio journal version 1 (Alluxio version >= 1.5.0) journal`